x/genutil/module.go

```diff
@@ -90,6 +90,6 @@ func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, data j
 
 // ExportGenesis returns the exported genesis state as raw bytes for the genutil
 // module.
-func (am AppModule) ExportGenesis(_ sdk.Context, _ codec.JSONMarshaler) json.RawMessage {
-	return nil
+func (am AppModule) ExportGenesis(_ sdk.Context, cdc codec.JSONMarshaler) json.RawMessage {
+	return am.DefaultGenesis(cdc)
```

Comment:
Shouldn't `ExportGenesis` return the list of GenTxs if exists, instead of the default state?

Reply:
No, since this is for exporting state. In other words, we already have initiated and bootstrapped the chain, so we don't need genesis txs to do so.